### PR TITLE
Newsletters: add 293 (2024-03-13)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Bitcoin Core PR Review Club @LarryRuane

### CODE REVIEW

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+## News
+
+- **Trustless onchain betting on potential soft forks:** ZmnSCPxj
+  [posted][zmnscpxj bet] to Delving Bitcoin a protocol for giving
+  control over a UTXO to a party that correctly predicts whether or not
+  a particular soft fork will activate.  For example, Alice thinks a
+  particular soft fork will activate and she's interested in acquiring
+  more bitcoins; Bob is also interested in acquiring more bitcoins but
+  does not think that fork will activate.  They agree to combine some
+  number of their bitcoins, at a ratio they both agree upon (e.g. 1:1),
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-13-newsletter.md:21]:
```suggestion
  amount of their bitcoins, at a ratio they both agree upon (e.g. 1:1),
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  so that Alice receives all of the combined bitcoins if the fork
+  activates by a certain time and Bob receives all of the combined
+  bitcoins if it does not.  If, before the deadline, a persistant
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-13-newsletter.md:24]:
```suggestion
  bitcoins if it does not.  If, before the deadline, a persistent
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  and a timeout.  A route for paying the invoice will be selected and a
+  [payment probe][topic payment probes] will be sent.  If the probe
+  sucessfully completes before the timeout is reached, the cost of using
+  the chosen route will be returned.  Otherwise, an error will be
+  returned.
+
+- [LND #8499][] makes significant changes to the Type-Length-Value (TLV)
+  types used for [simple taproot channels][topic simple taproot
+  channels] in order to improve LND's API to it.  We aren't aware of any
+  other LN implementation current using simple taproot channels, but if
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-13-newsletter.md:91]:
```suggestion
  other LN implementations currently using simple taproot channels, but if
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.1rc1][] is a release candidate for a maintenance release
+  of this predominant full node implementation.
+
+- FIXME:harding check for Bitcoin Core 27.0rc1
```
Comment (glozow) [inline _posts/en/newsletters/2024-03-13-newsletter.md:68]: https://bitcoincore.org/bin/bitcoin-core-27.0/test.rc1/ :)

**Review by murchandamus — COMMENTED** [review]
Reviewed but had no additional comments

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  number of their bitcoins, at a ratio they both agree upon (e.g. 1:1),
+  so that Alice receives all of the combined bitcoins if the fork
+  activates by a certain time and Bob receives all of the combined
+  bitcoins if it does not.  If, before the deadline, a persistant
+  chainsplit occurs where one chain activates the fork and the other
+  forbids it, Alice receives the combined bitcoins on the activated
+  chain and Bob receives the combined bitcoins on the
+  activation-forbidden chain.
+
+    The basic idea has been proposed before ([example][rubin bet]) but
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-03-13-newsletter.md:30]: Not sure if this is useful to point to, but Ill drop it here anyways:
https://bitcoinops.org/en/newsletters/2023/06/28/#speculatively-using-hoped-for-consensus-changes

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [LND #8136][] updates the `EstimateRouteFee` RPC to accept an invoice
+  and a timeout.  A route for paying the invoice will be selected and a
+  [payment probe][topic payment probes] will be sent.  If the probe
+  sucessfully completes before the timeout is reached, the cost of using
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-03-13-newsletter.md:84]:
```suggestion
  successfully completes before the timeout is reached, the cost of using
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  the chosen route will be returned.  Otherwise, an error will be
+  returned.
+
+- [LND #8499][] makes significant changes to the Type-Length-Value (TLV)
+  types used for [simple taproot channels][topic simple taproot
+  channels] in order to improve LND's API to it.  We aren't aware of any
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-03-13-newsletter.md:90]: Maybe?

```suggestion
  channels] in order to improve LND's API for it.  We aren't aware of any
```

**Comment by LarryRuane** [comment]
I know the review club section is light on questions (only 3), I'm having trouble finding questions that aren't too much in the weeds and also I was out of town all weekend. I have to step out for a few hours but will try to add more questions after I get back home at around 9PM MDT.

**Comment by LarryRuane** [comment]
@glozow - if you have time, please feel free to add review club questions or make any changes to the ones I just pushed, no need to check with me, thanks.

File: _posts/en/newsletters/2024-03-13-newsletter.md (hunk 5 above)
Comment (harding) [reply _posts/en/newsletters/2024-03-13-newsletter.md:30]: I think that's too different, so not including it, but thanks!

**Comment by harding** [comment]
Made edits and added lede and releases/RCs.  I didn't review added content or add topic links; I'll try to get to those in a few hours.

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+- [Core Lightning v24.02.1][] is a minor update to this LN node
+  containing "a few minor fixes [and] improvements in the cost function
+  of the routing algortithm."
+
+- [Bitcoin Core 26.1rc1][] is a release candidate for a maintenance release
+  of the network's predominant full node implementation.
+
+- [Bitcoin Core 27.0rc1][] is a release candidate for the next major
+  version of the network's predominat full node implementation.
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-13-newsletter.md:119]:
```suggestion
  version of the network's predominant full node implementation.
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+  and a timeout.  A route for paying the invoice will be selected and a
+  [payment probe][topic payment probes] will be sent.  If the probe
+  successfully completes before the timeout is reached, the cost of using
+  the chosen route will be returned.  Otherwise, an error will be
+  returned.
+
+- [LND #8499][] makes significant changes to the Type-Length-Value (TLV)
+  types used for [simple taproot channels][topic simple taproot
+  channels] in order to improve LND's API for it.  We aren't aware of any
+  other LN implementations current using simple taproot channels, but if
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-03-13-newsletter.md:142]:
```suggestion
  other LN implementations currently using simple taproot channels, but if
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning v24.02.1][] is a minor update to this LN node
+  containing "a few minor fixes [and] improvements in the cost function
+  of the routing algortithm."
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-03-13-newsletter.md:113]:
```suggestion
  of the routing algorithm."
```

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+- [LND #8136][] updates the `EstimateRouteFee` RPC to accept an invoice
+  and a timeout.  A route for paying the invoice will be selected and a
+  [payment probe][topic payment probes] will be sent.  If the probe
+  successfully completes before the timeout is reached, the cost of using
+  the chosen route will be returned.  Otherwise, an error will be
+  returned.
+
+- [LND #8499][] makes significant changes to the Type-Length-Value (TLV)
+  types used for [simple taproot channels][topic simple taproot
+  channels] in order to improve LND's API for it.  We aren't aware of any
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-03-13-newsletter.md:141]:
```suggestion
  channels] to improve LND's API for it.  We aren't aware of any
```

File: _posts/en/newsletters/2024-03-13-newsletter.md (hunk 10 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2024-03-13-newsletter.md]: Going to leave this since its a quote from somewhere else

Comment (bitschmidty) [reply _posts/en/newsletters/2024-03-13-newsletter.md]: Ok nvm, I see Dave already accepted this suggestion

File: _posts/en/newsletters/2024-03-13-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+  q0="What are the various conditions under which the execution of
+      `OP_CAT` may result in failure?"
+  a0="Fewer than 2 items on the stack, resulting item is too large,
+      disallowed by script verify flags (for example, the soft fork is
+      not activated yet), and appears in a non-taproot script (witness
+      version 0 or legacy)."
+  a0link="https://bitcoincore.reviews/bitcoin-inquisition-39#l-46"
+
+  q1="`OP_CAT` redefines one of the `OP_SUCCESSx` opcodes.
+      Why doesn't it redefine one of the `OP_NOPx` opcodes (which has
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-03-13-newsletter.md:78]:
```suggestion
      Why doesn't it redefine one of the `OP_NOPx` opcodes (which have
```

**Review by bitschmidty — COMMENTED** [review]
Reviewed PR review, topics, lede updates

**Pull request merged**